cmd/livecheck.rb

```diff
@@ -167,88 +254,178 @@ def print_latest_version(formula)
       end
 
       if Homebrew.args.json?
-        json_hash = {
-          "formula" => formula_name(formula),
-          "status"  => "skipped",
-        }
-        json_hash["messages"] = [skip_msg] unless skip_msg.nil? || skip_msg.empty?
-        return json_hash
+        if skip_msg.nil? || skip_msg.empty?
+          return status_hash(formula, "skipped")
+        else
+          return status_hash(formula, "skipped", [skip_msg])
+        end
```

Comment:
This suggestion is related to my comment about the `status_hash` method above. 
```suggestion
        return status_hash(formula, "skipped", [skip_msg])
```

Reply:
`skip_msg` defaults to an empty string (it's never `nil` despite that lingering `skip_msg.nil?` condition that we can probably remove), so there's the potential that the JSON output would include `"messages": [""]` unless we check for `skip_msg.empty?`.

The current JSON output only includes a message when it's not an empty string. I don't think there's value in an empty message string. There may be an argument for an empty messages array but that's better left for when we have the general discussion about the JSON output. For the purposes of this PR, I think it's reasonable to simply maintain the existing behavior.

Reply:
I pushed a commit that uses `return status_hash(formula, "skipped", (skip_msg.empty? ? nil : [skip_msg]))` instead (as it addresses my issue with using `[skip_msg]` when it could be `[""]`). It's a little more difficult to read but does this do it for you?